fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -769,10 +773,6 @@ public ChangedTransactionDetail reprocessTransactions() {
      *
      * If "suppliedTransactionDate" is not passed Id, the transaction date is set to the loans due date if the due date
      * is lesser than todays date. If not, the transaction date is set to todays date
-     *
```

Comment:
Was it incorrect or why did you remove?

Reply:
because the empty javadocs tags did not add any value imho